src/Parser.zig

```diff
@@ -3401,7 +3401,7 @@ fn findAggregateInitializer(p: *Parser, il: **InitList, ty: *Type, start_index:
 
         const field_count = struct_ty.data.record.fields.len;
         if (index < field_count) {
-            ty.* = struct_ty.data.record.fields[@intCast(usize, index)].ty;
+            ty.* = struct_ty.data.record.fields[@as(usize, @intCast(index))].ty;
```

Comment:
Ditto.

Reply:
`index` is `u64` so it's needed for 32-bit builds

Reply:
Right, forgot about that.

Reply:
`@as` should still be unnecessary.

Reply:
Removed this and the other one in `findScalarInitializer` and seems to have worked